### PR TITLE
Update 0419.lang

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
Update: Adding Localization - Enhanced Accessibility, Fixed typos.

### CODE REVIEW

File: Langs/0419.lang [hunk 1]
```diff
@@ -9,6 +9,8 @@ Medium=8.5
 Small=8.2
 Name=Arial
 [Strings]
+Enable enhanced accessibility?=Включить, "Улучшенную видимость"?
```
Comment (micwoj92) [inline Langs/0419.lang:12]: @Agc0de6 (lots of pings lately lol)
Do you approve this and below string?

Comment (Agc0de6) [reply Langs/0419.lang:12]: @micwoj92 just delete comma ok string 12 and that's all.

**Comment by Oleg-Chashko** [comment]
Removed comma.

**Review by Agc0de6 — APPROVED** [review]
Good.

**Pull request merged**